src/components/NavigationItem.svelte

```diff
@@ -25,6 +22,13 @@
 
   export default {
     computed: {
+      isBusy: item => {
+        if (!item.component) { return false }
+
+        if (item.component === 'storage') {
+          return !item.currentDiskUsage
```

Comment:
If the currentDiskUsage is 0 (which can be possible), the spinner will be continuously here even if the stack request succeeds. I think we should do: `return item.currentDiskUsage !== false` to check boolean type as well.